include/sharg/validators.hpp

```diff
@@ -479,37 +477,17 @@ class file_validator_base
  *
  * \remark For a complete overview, take a look at \ref argument_parser
  */
-template <typename file_t = void>
 class input_file_validator : public file_validator_base
 {
 public:
 
-    static_assert(std::same_as<file_t, void> || seqan3::detail::has_type_valid_formats<file_t>,
-                  "Expected either a template type with a static member called valid_formats (a file type) or void.");
-
     // Import from base class.
     using typename file_validator_base::option_value_type;
 
     /*!\name Constructors, destructor and assignment
      * \{
      */
 
-    /*!\brief Default constructor.
-     *
-     * \details
-     *
-     * If the class' template argument `file_t` names a valid seqan3 file type that contains a
-     * static member `valid_formats`, e.g. seqan3::sequence_input_file::valid_formats, then it generates the
-     * list of valid extensions from this file. Otherwise the extensions list is empty.
-     */
-    input_file_validator()
-    {
-        if constexpr (!std::same_as<file_t, void>)
-            file_validator_base::extensions = seqan3::detail::valid_file_extensions<typename file_t::valid_formats>();
-
-        file_validator_base::extensions_str = create_extensions_str();
-    }
-
     input_file_validator(input_file_validator const &) = default;             //!< Defaulted.
```

Comment:
```suggestion
    input_file_validator() = default;                                         //!< Defaulted.
    input_file_validator(input_file_validator const &) = default;             //!< Defaulted.
```

Reply:
Do we need this in the output_file_validator aswell?:
`output_file_validator() = default;                                          //!< Defaulted.`